green_mbtools/pesto/analyt_cont.py

```diff
@@ -8,7 +8,7 @@
 from .ac_utils import dump_input_caratheodory_data, load_caratheodory_data
 import green_mbtools.pesto.nevanlinna as nevan_exe
```

Comment:
We should remove the unused import statement here. Also, we might as well remove the entire Nevanlinna code from the repository.

Reply:
agree